tests/unit/DetailComment.spec.ts

```diff
@@ -84,7 +80,5 @@ describe('DetailComment', () => {
     // verify valid and input events
     expect(wrapper.emitted('valid').pop()[0]).toEqual(true)
     expect(wrapper.emitted('input').pop()).toEqual(['testing 4 5 6'])
-
-    wrapper.destroy()
```

Comment:
Is there a good reason for removing this? It was initially added to prevent a memory leak, but maybe is no longer needed.

Reply:
Actually destroy() is now changed to unmount(). Changed it accordingly.
Ref: https://test-utils.vuejs.org/migration/#destroy-is-now-unmount-to-match-Vue-3

Reply:
🤯 

I guess this will have to be changed everywhere. @leodube-aot @PaulGarewal @JazzarKarim, FYI.